src/formElementPlugins/listBuilder/src/displayComponents/ListInlineLayout.tsx

```diff
@@ -0,0 +1,117 @@
+import React, { useEffect, useState } from 'react'
+import { Button, Icon, Accordion } from 'semantic-ui-react'
+import { ApplicationDetails, ElementState, User } from '../../../../utils/types'
+import { ListLayoutProps } from '../types'
+import ApplicationViewWrapper from '../../../ApplicationViewWrapper'
+import SummaryViewWrapper from '../../../SummaryViewWrapper'
+import { buildElements, substituteValues } from '../helpers'
+import '../styles.less'
+
+const ListInlineLayout: React.FC<ListLayoutProps> = (props) => {
+  const { listItems, displayFormat } = props
+  return (
+    <>
+      {listItems.map((item, index) => (
+        <ItemAccordion
+          key={index} // Not ideal, but there isn't any other unique identifier
+          index={index}
+          item={item}
+          header={displayFormat.title}
+          {...props}
+        />
+      ))}
+    </>
+  )
+}
+export default ListInlineLayout
+
+// Inner component -- one for each Item in list
+const ItemAccordion: React.FC<any> = ({
+  item,
+  header,
+  index,
+  inputFields,
+  isEditable = true,
+  responses,
+  currentUser,
+  applicationData,
+  Markdown,
+  codes,
+  editItem,
+  deleteItem,
+  editItemText,
+  updateButtonText,
+  deleteItemText,
+  innerElementUpdate,
+  updateList,
+}) => {
+  const [open, setOpen] = useState(false)
+  const [edit, setEdit] = useState(false)
```

Comment:
From my understanding so far I think it would make more sense if using:
```suggestion
  const [isEditing, setIsEditing] = useState(false)
```

Reply:
And changes in everywhere using...

Reply:
Sure, good call: 532c123e